fdb-relational-core/src/main/java/com/apple/foundationdb/relational/recordlayer/metadata/RecordLayerSchemaTemplate.java

```diff
@@ -95,18 +95,22 @@ public final class RecordLayerSchemaTemplate implements SchemaTemplate {
     @Nonnull
     private final Supplier<String> transactionBoundMetadataSupplier;
 
+    private final boolean tablesAreIntermingled;
```

Comment:
Should we expose a method to probe the value of this flag from a `RecordLayerSchemaTemplate`? (Like the builder  method, `isIntermingleTables`?)

I might also prefer to keep the names consistent, though it's not like `tablesAreIntermingled` and `intermingleTables` are that hard to decipher as a pair. But `intermingleTables` is closer to the flag in the DDL